releases/8.0.0.md

```diff
@@ -77,37 +80,63 @@ JavaScript:
 
 1. In settings.py, add the following key to `DATABASES` to [improve indexing performance](https://github.com/archesproject/arches/issues/11382):
     ```
-        "OPTIONS": {
-            "options": "-c cursor_tuple_fraction=1",
-        },
+    DATABASES = {
+        "default": {
+            ...  # other entries 
+            "OPTIONS": {
+                "options": "-c cursor_tuple_fraction=1",
+            },
+        }
+    }
     ```
+1. Update `.gitignore`:
+    1. Remove `.tsconfig-paths.json` and `.frontend-configuration-settings.json`
+    2. Add `frontend_configuration`
 
-1. Update your frontend dependencies:
+1. Replace the `generate_frontend_configuration` import statement in `apps.py`:
     ```
-    rm -rf node_modules package-lock.json
-    npm install
+    from arches.settings_utils import generate_frontend_configuration
     ```
 
-1. Within your project, with your Python 3 virtual environment activated run:
-    ``` 
-    python manage.py migrate
-    ```
+    Should be updated to:
 
-1. Then run:
     ```
-    python manage.py updateproject
+    from arches.app.utils.frontend_configuration_utils import generate_frontend_configuration
     ```
 
-3. Create editable_future_graphs for your Resource Models using the command `python manage.py graph create_editable_future_graphs`. This will publish new versions of each Graph.
+1. Within your project, with your Python 3 virtual environment activated:
+    1.  Migrate your database:
+        ```
+        python manage.py migrate
+        ```
+
+    1. Then update your project:
+        ```
+        python manage.py updateproject
```

Comment:
I think it would be good to let people know that they can delete their `arches_urls.htm` file (I'd suggest using update project, but that might be a little intrusive)